libs/application/templates/social-insurance-administration/pension-supplement/src/lib/PensionSupplementTemplate.ts

```diff
@@ -0,0 +1,458 @@
+              {
+                onEvent: DefaultEvents.SUBMIT,
+                logMessage: coreSIAStatesMessages.additionalDocumentsAdded,
+              },
+            ],
+          },
+          lifecycle: pruneAfterDays(90),
+          roles: [
+            {
+              id: Roles.APPLICANT,
+              formLoader: () =>
+                import('../forms/AdditionalDocumentsRequired').then((val) =>
+                  Promise.resolve(val.AdditionalDocumentsRequired),
+                ),
+              read: 'all',
+              write: 'all',
+            },
+            {
+              id: Roles.ORGANIZATION_REVIEWER,
+              formLoader: () =>
+                import('../forms/InReview').then((val) =>
+                  Promise.resolve(val.InReview),
+                ),
+              write: 'all',
+            },
+          ],
+        },
+        on: {
+          SUBMIT: [{ target: States.TRYGGINGASTOFNUN_IN_REVIEW }],
+        },
+      },
+      [States.APPROVED]: {
+        meta: {
+          name: States.APPROVED,
+          status: 'approved',
+          actionCard: {
+            pendingAction: {
+              title: coreSIAStatesMessages.applicationApproved,
+              content: statesMessages.applicationApprovedDescription,
+              displayStatus: 'success',
+            },
+          },
+          lifecycle: DefaultStateLifeCycle,
+          roles: [
+            {
+              id: Roles.APPLICANT,
+              formLoader: () =>
+                import('../forms/InReview').then((val) =>
+                  Promise.resolve(val.InReview),
+                ),
+              read: 'all',
+            },
+          ],
+        },
+      },
+      [States.REJECTED]: {
+        meta: {
+          name: States.REJECTED,
+          status: 'rejected',
+          actionCard: {
+            historyLogs: [
+              {
+                // TODO: Þurfum mögulega að breyta þessu þegar við vitum hvernig TR gerir stöðubreytingar
+                onEvent: States.REJECTED,
+                logMessage: coreSIAStatesMessages.applicationRejected,
+              },
+            ],
+          },
+          lifecycle: DefaultStateLifeCycle,
+          roles: [
+            {
+              id: Roles.APPLICANT,
+              formLoader: () =>
+                import('../forms/InReview').then((val) =>
+                  Promise.resolve(val.InReview),
+                ),
+            },
+          ],
+        },
+      },
+    },
+  },
+  stateMachineOptions: {
+    actions: {
+      setApproveExternalData: assign((context) => {
+        const { application } = context
+        const { answers } = application
+
+        set(answers, 'approveExternalData', true)
+        return context
+      }),
+      /**
+       * Copy the current answers to temp. If the user cancels the edits,
+       * we will restore the answers to their original state from temp.
+       */
+      createTempAnswers: assign((context, event) => {
```

Comment:
Why is this needed? Dont the answers get assigned to the application object?

Reply:
@karenbjorg im happy to approve but I would love an answer to why this is needed?